ndcube/extra_coords/table_coord.py

```diff
@@ -136,7 +140,7 @@ def _generate_tabular(lookup_table, interpolation='linear', points_unit=u.pix, *
         raise TypeError("lookup_table must be a Quantity.")  # pragma: no cover
 
     ndim = lookup_table.ndim
-    TabularND = tabular_model(ndim, name=f"Tabular{ndim}D")
+    tabular_nd = tabular_model(ndim, name=f"Tabular{ndim}D")
```

Comment:
`tabular_model` returns a class object so this should be caps.
```suggestion
    TabularND = tabular_model(ndim, name=f"Tabular{ndim}D")
```